runtime/src/lib.rs

```diff
@@ -223,10 +210,28 @@ pub mod opaque {
 		}
 	}
 
-	// todo: check possibililty of adding the beefy ecdsa public
```

Comment:
isn't this todo still valid?

Reply:
There's a separate pr. Will update comments.